\section{Discussion}
\label{sec:discussion_ks}

The KS-Burden test was able to outperform SKAT-O under all simulated scenarios.
Furthermore, the KS-Burden test performed better than the classical burden tests in all but one case.
This demonstrates that the combination of KS and classical burden test is able to provide potentially greater statistical power when considering clusters of disease causing mutations.
The simple KS test performed well in scenarios which contained mostly neutral variants and was even able to perform equally well as SKAT-O in scenarios in which more then 2/5 of all included variants were neutral, suggesting that the KS test is able to detect a disease causing gene in situations in which the causal cluster is large.
This is important since it is common practice to filter out potential neutral genetic variants via bioinformatic annotations which can lead to a significant increase in causal cluster size.

As expected, the KS test loses considerable statistical power when the proportion of causal variants increases.
Nevertheless, the combined KS-Burden test is able to compensate these shortcomings by combining it with the classical burden test statistic while assuming independency of the two tests.
While simulations are unable to prove independence of the two test statistics, these simulations represent realistic and practical scenarios and are suggesting that in practice both tests act independently.

Interestingly, the performance of both KS and SKAT are rather similar.
Both tests have the greatest statistical power in situations in which most variants are neutral but lose statistical power given a large proportion of causal variants.
However, while SKAT is able to maintain some degree of statistical power in situations which have a large percentage of causal mutations, the KS test loses nearly all statistical power in these scenarios.
In contrast, the KS test is able to perform better in the presence of a small causal cluster.

However, a benefit of the KS test is its simplicity.
Other spatial approaches to genomic data, most notably IL-K~\cite{Ionita-Laza2012}, KERNEL~\cite{Schaid2013} and CLUSTER~\cite{Lin2014}, make use of clustering methods.
These methodologies are commonly more complex than the KS test as well as computationally intensive.
In contrast, the KS test is simpler while providing considerable statistical power in situations with a clear causal cluster.

The classical burden and CMC test outperform all other statistical approaches when all genetic variants are causal.
This is not surprising given that the burden test assumes that all genetic variants have the same direction of effect.
Furthermore, burden and CMC are equivalent in most scenarios since the CMC tests the proportion of cases vs controls which have at least one mutations, while the burden tests the absolute difference in number of  mutations between the two groups of subjects.
Given large enough effect size, the burden and CMC will be equivalent.

Gene-based analysis of rare variants within the UK Biobank suggests no genome-wide significantly associated gene.
There are a number of possible reasons for these null findings.
First, selected variants were chosen from a MAF band and did not included private mutations (mutations specific to a single individual).
While most sequencing-based studies are able to detect a number of genetic mutations previously unobserved, microarray data makes use of imputation in order to obtain genotypes of rare variants.
While this method works well for an MAF above 0.01\% it is unable to recover private mutations.
Second, aggression is a common trait and all humans express some form of aggressive behavior (see Chapter~\ref{cha:introduction}).
While rare variants have been shown to  influence a number of common traits, such as height~\cite{Marouli2017}, no such findings have been made for behavioral traits~\cite{Chabris2015}.
Results from the present rare variant analysis on impulsive aggression are therefore in line with these previous null findings.

Furthermore, it is important to note a few limitations of this study.
An obvious limitation of the KS test is its inability to detect more than one causal cluster.
A possible solution for this limitation would be to consider the sum of the absolute difference between the two empirical cumulative distribution functions. 
However, this would remove the option to clearly identify the location of a region within a gene. 
Furthermore, the current form of the KS does not allow the inclusion of variant weights.
For example, one might  weight specific variants higher than others depending  on their corresponding deleterious score or MAF\@.
Nevertheless, it is possible to include weights within the KS test by re-ordering variants based on their weights in exchange for the loss of location information.
Finally, there is  uncertainty about the independence between the KS and burden tests.
However, the  present results suggest that the two tests act independently in most real world scenarios.  


