\section{Method}
\label{sec:method}

Assuming $p$ variants and a dichotomous phenotype of $n$ subjects was acquired.
For the $ith$ subjects lets denote $\pmb{G_i} = (G_{i1}, G_{i2},\ldots , G_{ip})$ and $y_i$ as the genotype as well as the phenotype respectively.
Further I assumed an additive genetic model and let $G_{ip} = 0, 1, 2$ represent the number of rare alleles at variant $p$.  

One can then calculate the empirical cumulative distribution function (ecdf) for both cases ($F_{A(G)}$) and controls ($F_{U(G)}$) or any other dichotomous phenotype.
The ecdf is a monotonic step function, increasing from $0$ at the first position of the gene, to $1$ at the last position of the gene.
For $p$ rare variants occurring at positions  $G_{i1}, G_{i2}, \ldots, G_{ip}$, the empirical distribution function of $G$ is given by

\begin{equation}
  F(x) = \frac{1}{p}\sum^p_{i=1}I(X_i \leq x)
\end{equation}

in which $I(x)$ is the indicator function for a fixed given variant $x$.
The KS statistic $K$ is defined as the supremum of the absolute difference between the two empirical distribution functions.

\begin{equation}
	K_{KS} = \sup_x | F_A(x) - F_U(x) |
\end{equation}

While the KS test is distribution free when $F$ is continuous, this is not the case with discrete data, such as allele count.
Indeed, application of the Kolmogorov distribution to obtain critical values for $K$ for discrete data yields conservative estimates~\cite{Walsh1963,Conover1972}. 
\citet{Conover1972} proposed an alternative way to obtain critical values given a hypothesized discrete distribution function.
However, this approach is not feasible with samples sizes $>30$.
Therefore, I applied a permutation based approach to estimate p-values as 
\begin{equation}
  p_{per} = \frac{\sum^b_{b=1} I(K_b \geq k)+1}{b+1}
\end{equation}
in which $K_b$ is the test statistic of the $b^{th}$ permuted sample.

\subsection{Omnibus Test for KS and Burden}
\label{sub:omnibus_test_for_ks_and_burden}

The KS test makes the strong assumption that in a given genomic region only a certain causal cluster of rare variants are related to the phenotype in question.
This assumption does not contradict that of the burden test which assumes that all variants under investigation have the same direction of effect and can be seen as two sides of the same coin.
Thus it might be desirable to combine the two test statistics in order to obtain a combined test statistic.

I define the test statistic of the burden test as 
\begin{equation}\label{eq:burden_simple}
  K_{Burden} = (\sum^p_{i=1} (\pmb{y}-\bar{y}) \times \pmb{G_i})^2
\end{equation}
This is equivalent to the form described in Section~\ref{sub:burden_test}.

One can combine the two tests by Fischer's method 
\begin{equation}
	\chi^2_4 \sim - 2 (\ln(p_{KS}) + \ln(p_{Burden}))
\end{equation}
in which the test statistic $\chi^2$ follows a $\chi^2$-distribution with $4$ degrees of freedom.
I have choose to call this combined test KSburden.

Implementation of the KS, Burden and KSburden was done in C++ and can be found at \url{https://github.com/rmporsch/ksburden}.
This repository also includes associated scripts and programs to repeated simulations described within this manuscript.

\subsection{Simulation Study}
\label{sub:simulation_study}

In contrast to most previous simulation studies investigating the statistical properties of rare variant association tests I made use of actual sequencing data.
Thus, genotypes were not simulated but acquired from previous association studies.
The main aim of using real data is to accurately reflect the diversity of genes within the human genome as well as limitations commonly encountered in sequencing based studies.
Most gene based tests are confronted with a variety of small and medium gene lengths which is often not reflected in the original power estimations.
Hence I will make use of a large sequenced sample to investigate the statistical properties of the KS and KSburden test.

\subsubsection{The Seed population: The Hirschsprung data}
\label{ssub:The_Seed_population:_The_Hirschsprung_data}

Hirschsprung's disease is a rare congenital disorder occurring in about one of $5,000$ births~\cite{Amiel2001}.
The here used sample contains X cases as well as XX controls.

Further, only non synonymous exonic variants (stopgain, missense, startloss, stoploss) with an MAF $\leq 1\%$ were included in the analysis. 
Overall frequency of rare non synonymous mutations are displayed in Figure~\ref{fig:num_var} and show great variations.
In general, most genes have only a few rare variants and only a few large genes have more than 100 rare genetic mutations. 

\begin{figure}[htpb]
  \centering
  \includegraphics[width=0.6\linewidth]{figures/number_var_genes.pdf}
  \caption{Overall frequency of rare non-synonymous mutations.
    The range of rare variants per gene ranged from $3$ to $491$ with a median of $5$.
  }\label{fig:num_var}
\end{figure}

\subsubsection{Simulation Framework}
\label{ssub:Simulation_Framework}

From the set of $16,510$ genes I selected 50 genes at random which had at least 3 or more rare variants. 
I defined mutation as rare if the minor allele frequency (MAF) was below 1\%. 
Within $p$ rare variants of a gene I assigned causal status to $p_1 \ldots p_t$ positions, in which $t$ represents the total number of causal variants.
For simplicity I assigned the causal cluster at the beginning of the gene.
Within these simulations I gradually increased the size of the causal cluster to eventually cover the whole gene. 

The phenotype was simulated via a liability threshold model.
Hence the phenotype $Y_i$ of the $i^{th}$ subject was generated via
$Y = G\times E' + \epsilon$
in which $G$ is the standardized genotype matrix of $n=1000$ subjects with $p$ variants.
$E$ is the effect size vector of size $1\times p$ and $\epsilon$ is a standard normal distributed error term with a mean of $0$ and a standard deviation of $\sqrt{1-h^2}$, in which $h^2$ is the assumed heritability or the genetic effect on the liability distribution.
The effect $h$ was uniformly distributed across all causal variants.
I assigned case status for each subjects whose $Y_i$ is above a certain liability threshold $q$.
Any subject above $q$ was assigned case status, while the remaining subjects were deemed to be controls.
This process was repeated until $500$ cases and an equal number of controls were collected.

\subsubsection{Application of the KSburden}
\label{ssub:Application_of_the_KSburden}

The KSBurden test was further applied to the UK BioBank to investigate distributional differences of rare variants between aggressive and less aggressive individuals.
As described in the previous chapter the UK BioBank is a chip-array data set.
Commonly genotyped data is not applicable for rare variant testing due to the low imputation quality of low frequency variants.
However, given a very large sample size, imputation quality is gradually improving thus allowing investigations of rare variants.
Indeed, the UK BioBank is with over $150,000$ samples relatively large.
As can be seen in Figure~\ref{fig:imputation}, imputation quality is acceptable for variants with a frequency between 1\% and 0.1\%, given the commonly used quality score cut-off of $0.2$.
Therefore, the exceptional imputation quality of the UK BioBank data allows to perform gene based rare variant tests on array data.

\begin{figure}[htpb]
  \centering
  \includegraphics[width=0.8\linewidth]{example-image-a}
  \caption{Imputation Quality of the UK BioBank}\label{fig:imputation}
\end{figure}
