\section{Method}
\label{sec:method}

Considering a gene with the positions of nucleotides labeled numerically $1, 2, \ldots p$, in which the position of a rare variant is regarded as a random variable, $X$.
The rare variants observed in each group (cases or controls) are considered a realizations of $X$.
One can then calculate an empirical cumulative distribution function (ecdf) for each group ($F_{A(X)}$ for cases, $F_{U(X)}$ for controls), which is a monotonic step function, increasing from $0$ at the first position of the gene, to $1$ at the last position of the gene.
For $\gamma$ rare variants occurring at positions $X_1, X_2, \ldots, X_p$, the empirical distribution function of $X$ is given by

\begin{equation}
  F(x) = \frac{1}{\gamma}\sum^p_{i=1}I(X_i \leq x)
\end{equation}

in which $I(x)$ is the indicator function for variant $x$.
The KS statistic $K$ is defined as the supremum of the absolute difference between the two empirical distribution functions.

\begin{equation}
	K_{ks} = \sup_x | F_A(x) - F_U(x) |
\end{equation}

While the KS test is distribution free when $F$ is continous, this is not the case with descrete data such as allele count.
Indeed, application of the Kolmogorov distribution to obtain critical values for $K$ for discreate data yiels conservative estimates~\cite{Walsh1963,Conover1972}. 
\citet{Conover1972} proposed an alternative way to obtain critical values given a hypothesized discrete distribution function.
However, this approach is in general not feasible with samples sizes $>30$.
Therefore, I applied a Monte Carlo based approach to estimate p-values as 

\begin{equation}
  p_{per} = \frac{\sum^b_{b=1} I(K_b \geq k)+1}{b+1}
\end{equation}

in which $K_b$ is the test statistic of the $b^{th}$ simulation sample.
It is important to note that if $\gamma$ is large and distributred among $p$ variants in a continous fashion $F$ can be considered as continues therefore reducing the necessarity to perform this computationaly intesive approach.
However, given the current sample sizes in sequencing based association studies this seems an unlikely scenario.

\subsection{Omnibus Test for KS and Burden}
\label{sub:omnibus_test_for_ks_and_burden}

The KS test makes the strong assumption that in a given genomic region only a certain causal cluster of rare variants are related to the phenotype in question.
This hypothesis does not contradict that of the burden test which assumes that all variants under investigation have the same direction of effect and can be seen as two sides of the same coin.
Thus it is desirable to combine the two test statistics in order to obtain a combined test statistic.

I define the test statistic of the burden test as 
\begin{equation}\label{eq:burden_simple}
  K_{Burden} = {(\sum^\gamma_{i=1} (y-0.5) \times G_i)}^2
\end{equation}

in which $y$ is the vector of case-control status and $G$ the genotye matrix of size $n \times p$, in which $n$ is the number of samples.
This is equivalent to the form described in Section~\ref{sub:burden_test}.

\subsubsection{Relationship between KS and Burden Test}
\label{ssub:Relationship_between_KS_and_Burden_Test}

In order to combine the two test it is desirable to explore the relationship between the two tests.
In contrast to the KS test, the Burden test makes use of the sum of all alleles in cases and controls respectivly alone.
Hence we can rewrite the Equation~\ref{eq:burden_simple} as
\begin{equation}
  K_{burden} =  |\pmb{1}X_{A}  - \pmb{1}X_{U}|
\end{equation}
Lets assume that $\pmb{1}X_{A} = \pmb{1}X_{U}$ then we can rewrite the KS test as
\begin{equation}
  \begin{aligned}
    K_{ks} & = \sup |F_{A(X)} - F_{U(X)}|  \\
           & = \frac{1}{\pmb{1}X} \sup |(\pmb{1}X^{(A)}_i - \pmb{1}X^{(U)}_i)|
  \end{aligned}
\end{equation}
Therefore, if $K_{ks}=0$ then $K_{Burden}$ must be $0$, suggesting that under the null of both KS and Burden the tests are not strictly independent.
However, under the null of the burden test alone, given $\pmb{1}X_{A} = \pmb{1}X_{U}$, then $K_{Burden} = 0$ but the test statistic of the KS test is not necessarily $0$.
Thus, the two tests can be consindered dependent if $K_{ks} = 0$.
In the next section I will demonstrate that this is an unlikely scenario and that in pracitce the two tests can be seen as independent.

Hence one can combine the two tests by Fischer's method 
\begin{equation}
	\chi^2_4 \sim - 2 (\ln(p_{KS}) + \ln(p_{Burden}))
\end{equation}
in which the test statistic $\chi^2$ follows a $\chi^2$-distribution with $4$ degrees of freedom.
I have choose to call this combined test statistic KSburden.

Implementation of the KS, Burden and KSburden was done in C++ and can be found at \url{https://github.com/rmporsch/ksburden}.
This repository also includes associated scripts and programs to repated simulations described within this manuscript.

\subsection{Simulation Study}
\label{sub:simulation_study}

In contrast to many other simulation studies on gene based tests I used real sequencing data in order to form the basis for my simulations.
The main aim of using un-simulated data is to accuratly reflect the diversity of genes within the human genome.
Most gene based tests are confronted with a varierty of small and medium gene lengths which is often not reflected in the original power estimations.
Hence I will make use of a large sequenced sample to investigate the statistical power of the KS and KSburden test.
In the next section I will describe the sample used in my simulations as well as the simulation framework in general.

\subsubsection{The Seed population: The Hirschsprung data}
\label{ssub:The_Seed_population:_The_Hirschsprung_data}

\subsubsection{Simulation Framework}
\label{ssub:Simulation_Framework}

From the set of XXX genes I selected 50 genes at random which had at least 3 or more rare variants. 
I defined mutation as rare if the minior allele frequency (MAF) was below 1\%. 
Within $p$ postion of a selected gene I assigned causal status to $p_1 \ldots p_t$ positions, in which $t$ represents the total number of causal variants.
For simplicity I assigned the causal cluster at the beginning of the gene.
Within these simulations I gradually increased the size of the causal cluster to eventually cover the whole gene. 

The phenotype was simulated via a liability threshold model.
Hence the phenotype $Y_i$ of the $i^{th}$ subject was generated via
$Y = G\times E' + \epsilon$
in which $G$ is the standardized genotype matrix of $n=1000$ subjects with $p$ variants.
$E$ is the effect size vector of size $1\times p$ and $\epsilon$ is a standard normal distributed error term with a mean of $0$ and a standard deviation of $\sqrt{1-h^2}$, in which $h^2$ is the assumed heritability or the genetic effect on the liability distribution.
The effect $h$ was uniformly distributed across all causal variants.
I assigned case status for each subjects whose $Y_i$ is above a certain liability threshold $q$.
Any subject above $q$ was assigned case status, while the remaining subjects were deemed to be controls.
This process was repeated until $500$ cases and an equal number of controls were collected.

\subsubsection{Application of the KSburden}
\label{ssub:Application_of_the_KSburden}

I will further use the UKBioBank data to apply the KSBurden test to investigate distributional differences of rare variants between aggressive and less aggressive individuals.
As decribed in the previous chapter the UKBioBank is a chip-array data set.
Commonly genotyped data is not applicable for rare variant testsing due to the low imputation quality of low frequency variants.
However, given a very large sample size, imputation quality is gradually improving.
Therefore, allowing investigations of rare variants.
Indeed, the UKBioBank is with over $150,000$ samples relatively large.
As can be seen in figure~\ref{fig:imputation}, imputation quality is acceptable for variants with a frequency between 1\% and 0.1\%, given the commonly used quality score cut-off of $0.2$.
Therefore, the execptional imputation quality of the UKBioBank data allows to perform gene based rare variant tests on array data.

\begin{figure}[htpb]
  \centering
  \includegraphics[width=0.8\linewidth]{example-image-a}
  \caption{Imputation Quality of the UKBioBank}\label{fig:imputation}
\end{figure}

In the next section I will describe the result of my simulations as well as show the effectiveness of the KS test on the UKB\@.
