\section{Methods}
\label{sec:methods}

\subsection{UK BioBank}
\label{sub:uk_biobank}
The UK BioBank~\cite{Allen2014} is a population based study of people aged 40--69 years old. 
Participants were recruited between 2006 and 2010 via mail invitation (response rate $5.47\%$)~\cite{Sudlow2015} and were assessed with the help of self-report questionaires as well as other anthropometirc assessments.
Written informed consent was aquired from all participants and the study was approved by National Research Ethics Service Committee North Westâ€“Haydock, UK\@.

\subsection{Phenotype}
\label{sub:phenotype}

Within this dataset the two variable of intestest are impulsive aggression and risk taking.
Phenotypical details of  Alcohol consumption, Neuroticism, and smoking can be found in the appendix of this chapter. 

\subsubsection{Impulsive Aggression}
\label{ssub:impulsive_aggression}
Impulsive aggression was measured with a single item question.
Participants were asked to answer the following question:
\begin{displayquote}
  Have you ever had a period of time lasting at least two days when you were so irritable that you found yourself shouting at people or starting fights or arguments?
\end{displayquote}
Table~\ref{tab:descriptive_gwas} shows the number of participants for both Caucasian and non-Caucasian samples.
The phenotype has a relatively high missingness rate within analysed subjects ($67.984\%$) with $40,861$ subjects anwered with `no' wile $9,397$ anwered with `yes'.

\subsubsection{Risk Taking}
\label{ssub:risk_taking}
Risk taking is a single item instrument which subjects answer with either yes or no.
Participants were asked to answer the follwing quesion:
\begin{displayquote}
  Would you describe yourself as someone who takes risks?
\end{displayquote}
In contrast to the previous described question, risk taking has a relatively large sample size and low missingness rate ($3.81\%$).
In total $107,011$ subject answered with `yes', while $39,435$ answered with `no' (see Table~\ref{tab:descriptive_gwas}).

\begin{table}[!htpb]
	\centering
	\resizebox{\textwidth}{!}{\input{ukb_assoc/tables/descriptive.tex}}
  \caption{Sample size and missingness across Caucasians and non-Caucasians }\label{tab:descriptive_gwas} 
\end{table}

\subsection{Genetic analysis}
\label{sub:genetic_analysis}
This study made use of the imputed genetic data of the UK BioBank, compromising around $\sim73$ million SNPs, short indels and larger structural variations of $152,249$ subjects.
Individuals were genotyped on two different custom genotype arrays, namely the UL BiLEVE as well as the UK Biobank Axiom array from Affymetrix. 
UL BiLEVE was used for most samples ($\sim100,000$), while the array chip from Affymetrix, which was optimsied to support genotype imputation, was used for the remaining participants. 
Quality control and imputation was performed central with the UK BioBank.
However, I further conducted additional quality control.
Variants which whose missing call rate did exceed 10\% as well as those with minor allele frequency below 1\% were exlcluded.
Further variants which did not pass a Hardy-Weinberg equilibrium test threshold of $1e-9$ were also removed from the analysis.
In addition I only included participants with `white European' ancestry identified by a k-mean cluser of the first four principle components of genotype data.
The total number of participants remaining after quality control of geno- and phenotype are displyed in Table~\ref{tab:descriptive_gwas}.

Assocation analysis of autosomal SNPs was performed with Plink~\cite{Purcell2007,Chang2015} with age, sex, genotype array chip and the first 10 principle components as covariants.

\subsection{Identification of lead SNPs}
\label{sub:Clumping}

Lead SNPs were identified using clumping.
Clumping is the process of grouping SNPs within a given genomic window and LD together.
Various thresholds and window sizes can be used.
Within this study the more commonly $250kb$ window region was used.
SNPs were consindered to be within the same LD group when passing an threshold of $r^2 \ge 0.2$.
The SNPs with the loweset p-value within each group was considered the lead SNPs.
Lead SNPs were used for the computation of polygenic risk scores as well as for the GWAS Cataloge lookup (see Section~\ref{sub:GWAS_Cataloge_Lookup}).

\subsection{GWAS Catalog Lookup}
\label{sub:GWAS_Cataloge_Lookup}

The GWAS catalog~\cite{Welter2014} is a publicly available resource and contains genome wide significant associated SNPs in a variety of different populations.
The catalog was used to look up potential association of lead SNPs in other phenoypes.
SNPs in the catalog and lead SNPs in each of the two GWASs were matched if both were within a $250kb$ window and in LD ($\ge 0.2$) of each other (see Section~\ref{sub:Clumping}).
Direction of effect within the cataloge was infered from the provided test statistics or effect sizes given availability.

\subsection{Polygenic Risk Score}
\label{sub:prs}

Polygenic Risk Scores (PRS) summaries the genetic risk from multiple loci for each individual.
These risk scores can be used to predict the genetic risk of an individual for a specific trait.
In general PRS are the sum of the weighted genetic loci of each study participant and can be expressed simply as
\begin{equation}
	\kappa_n = \sum^p_{j=1} \beta_j X_{nj}
\end{equation}
in which $\beta$ is the regression coefficients for each genetic loci $j$ and $X$ is the genotype matrix of size  $n\times p$ with $p$ loci of $n$ subjects.

Multiple different methods have been devloped to compute PRS\@.
Within this study we made use of \textit{lassosum}.
A recently develped PRS-framework, which I co-developed, to estimate polygenic scores~\cite{Mak2016}.
\textit{Lassosum} makes use of a penalized regression to estimate PRS from summary statistics.
The usage on summary statistics alone makes it easier to compute PRS scores for a multitude of phenotypes on large scale data.
Within the estiamtion of PRS only lead SNPs were used.

\subsection{Conditional False Discovery Rate}
\label{sub:conditional_false_discovery_rate}

Pleiotropy informed Conditional False Discovery Rate (cFDR) was used to identify additional genetic loci associated with risk taking or impulsive aggression. 
In principle, cFDR makes use of pleiotropic effect between two phenotypes to increase statistical power.
Within this chapter I computed the emprical culmulative distribution function on nominal p-values of phenotype A conditional on p-values of phenotype B passing a given threshold~\cite{Andreassen2013}.
Further applied genomic control to all nominal p-values using the genomic inflation factor $\lambda{GC}$ to all nominal p-values.
This was done by converting p-values to z-scores, estimating the median z-score for each GWAS, and deviding the squared median z-score by the expected median of a $\chi^2$-distribution  with $df=1$.

Conditional QQ-plots were constructed by plotting adjusted nominal p-values on the y-axis, while the emprical distribution function of p-values passing $p\leq0.1, p\leq 0.01$ or $ p\leq 0.001$ on the x-axis.

In addition, to conditional QQ plots I also made use of the conditional FDR (cFDR) which is defined as~\cite{Andreassen2013}
\begin{equation}
  FDR(p_1|p_2)=\pi_0(p_2)p_1/F(p_1|p_2)
\end{equation}
in which $p_1$ and $p_2$ are the p-values for the first and second phenotype.
The function $F(p_1|p_2)$ is the conditional cdf which can be replaced by the conditional emprical distribution function.
Further, $\pi_0(p_2)$ is the assumed conditional propotion of null SNPs for phenotype 1 given that that its p-values are smaller or equal that of the second phenotype.
Within this study I followed a conservative approach by setting $\pi(p_2)=1$~\citet{Andreassen2013}.
A SNP was considered of interest when its conditional FDR $\leq 0.01$.

\subsection{Genetic Correlation}
\label{sub:genetic_correlation_ukb_assoc}

Computation of genetic correlations were done with LD-score (see Secton~\ref{sub:ld_score_regression}).
Thus, pairwise genetic correlations were computed on the summary statistics of each GWAS\@.
Multiple testing was addressed with the more stringend Bonferroni correction.
