\section{Methods applied in genetic studies on humans}
\label{sec:methods_applied_in_genetic_studies_on_humans}

The investigation of the underlying gentic architecture of human traits is limited to non-experimental studies.
Thus previous research has made use of population based studies to improve our understanding of underlying genetic components.
One can broadly distinquish between two types of genetic studies on humans.
Before the availability to identify specific genetic mutation within the human genome, researchers have make use of twins.
Specifically, monozygotic and dizygotic twin pairs provide the unique ability to easaly estimate the contribution of genetic and enviormental components on a trait.
However, more recently new molecular methods have been developed to identify genetic variations within the human genome.
This has allowed researchers to identify specific genetic markers associated with a variety of human traits and diseases.

Within this section I will describe commonly used methods in both twin and molecular based association studies.
I will further describe methods used to estimate heritability and genetic correlations in association studies.

\subsection{Twin based studies}
\label{sub:twin_based_studies}

Twins have always been of special interest to scholor.
Indeed already the famous physician Hippocrates has been reported to be interested in twins (5th century BCE).
The Roman politician and author Cicero described Hippocrates's observations of two ill brothers, suspected to be twins, with similar identical disease progression~\cite{Cicero44BC}.
Much later Francis Galton was one of the first persons to use twins in order to investigate the effect of genes and the enviornment on human behavior~\cite{Rende1990} and with the discovery of two distinct types of twins, namely monozygotic and dizygotic twins, by~\citet{Simens1924}, twin studies became an estaiblised instrument in investigating genetic factors in humans.
\acrfull{mz} twins develop from a single fertilized egg and therefore share all of their genetic variations.
On the other hand, \acrfull{dz} twins (DZ) develop from two fertilized eggs and therefore share only 50\% of their genetic variations.

This distinction forms the basis of all twin studies and allows to state structural equations relating observed trait and theorized underlying genetic and enviormental effects.
The genetic effects can be further distiguished between additive genetic effects (A) which represents the accumulated effect of all genetic variations and non-additive effects which represents interaction on the same genetic locus, also called dominance (D), and different loci (epistasis).
Enviornmental components are differentiated into shared enviornment (C) and unique enviornment (E).
Therefore the total variance of a any particular trait P is the sum of A, D, C and E.

Since one can assume different correaltions between MZ and DZ twins one can estimate components of P.
While the correaltions between twins within C and E are the same in both MZ and DZ twins, namly $1$ and $0$ respectivly.
However, MZ twins have a correlation of $1$ for both A and D, while DZ pairs have a correlation of $\frac{1}{2}$ and $\frac{1}{4}$ respectivly.
Therefore differences within MZ twins can be attributed to E alone.
Further, when we assume that DZ and MZ twins are exposed to the same degree of similarity within their enviornment, the differences in similarity between MZ and DZ twins is an estimate of A.
This is also called Falconer's formula (see Formula~\ref{eq:falcon}) and can be used to estimate heritability, or the estimated of the relative importance of genetic effects.

\begin{align}
  h^2 &= 2(r_{MZ}-r_{DZ})\label{eq:falcon} \\ 
  C &= r_{MZ} - h^2 \nonumber \\
  E &= 1 - h^2 + c^2 \nonumber 
\end{align}

However, while the above is attractive it its simplicity, today's twin studies use structural equation models to model genetic and enviornment effects.
\acrfull{sem} are more flexible in modeling specifc hypothesis, are able to test for sex differences as well are able to handle multivariate data~\cite{Rijsdijk2002}.
Figure~\ref{fig:ace} displays a classical twin model.
Additive genetic (A), common enviornment (C) and unique enviornment (E) are so called latent variables.
Latent variables are not directly observed variables but are inferred from observed ones.
Observed variables are commonly displayed in cornered boxes.
The causal paths $a$, $c$, and $e$ represents the to estimate effect of the components on the trait T.
The sqaure or these estimates are the variance components of A, C and E respectivly.
The double headed arrows represents the correlations among A and C.

\begin{figure}[htpb]
  \centering
  \scalebox{0.6}{\input{introduction/figure/ACE.tex}}
  \caption{
    Basic ACE model.
    This basic model contains the latent variables A, C and E for twin 1 and 2, as well as the observed variable T with the mean $\mu$.
  }\label{fig:ace}
\end{figure}

Thus one can also write the total covariance of MZ and DZ twin pairs as
\begin{equation}
  cov(MZ) = 
  \begin{pmatrix}
    a^2 + c^2 + e^2 & a^2 + c^2 \\
    a^2 + c^2 & a^2 + c^2 + e^2
  \end{pmatrix}
\end{equation}
and 
\begin{equation}
  cov(DZ) = 
  \begin{pmatrix}
    a^2 + c^2 + e^2 & \frac{1}{2}a^2 + c^2 \\
    \frac{1}{2}a^2 + c^2 & a^2 + c^2 + e^2
  \end{pmatrix}
\end{equation}

Modern SEM software is able to estiamte these parameters by minimising the a goodness-of-fit statistic between the observed and predicted covariance matrices.
Most commonly this is done via a maximum-likelihood function.
Further the overall goodness-of-fit of the model relativly to a perfect fit, meaning that parameters are considered as `free' and their maximum-likelihood estiamte will equal the sample covariance, are measured by a likelihood raio sqaure statistic ($\chi^2$).
Therefore, should we fail to reject the null hypothesis that our model in Figure~\ref{fig:ace} is different from a perfect fitted model we have reason to assume that our genetic model fits the data.

The use of SEM allows for great flexiblity and a variaty of models to be estimated.
In the past few decades numerous twin studies on a variety of traits have been performed.
It not only allowed us to test for differences in the genetic architecture between the sexes but also look at how the influence of genetic factors change over age.
However, due to the new advancedment in acquiering genetic information of individual more research has been shifted to \acrfull{gwas}.
Hence, in the next section I will outline the methods applied GWAS\@.

\subsection{Association studies of common variants}
\label{sub:association_studies_of_common_variants}

Large scale genomic assocation studies have enabled researchers to investigate specific genetic factors associated with a certain trait.
Hence, while twin studies were only able to give an estimate of the total contribution of genetic factors on a phenotype, assocation studies are able to elucidate the specific molecular basis of complex traits.
Specifically, these genome wide association studies untilize single \acrfull{snp}, to identify specific genetic markers associated with a certain trait.
SNPs are variations within the genome at a specifc postion and underly differences in traits and disease susceptibility. 

Before explaining assocation studies in more detail it is important to mention the comcept of \acrfull{ld}.
LD is `the nonrandom assocation of alleles at different loci'~\cite{Slatkin2008} and forms a marker of the population genetic mechanism that is at play within our genome.
For example, two loci are said to be in high LD when allele $A$ at one loci co-occures with allele $B$ at a different loci at a higher frequency then you would expect if the two loci were independent.
Hence the level of LD can be quantified as $D_{AB}=p_{AB} - p_{A}p_{B}$ in which in which $p_{AB}$ is the frequency that $A$ and $B$ occure together wile $p_A$ and $p_B$ is the frequency of $A$ and $B$ respectivly.
Nevertheless, $D_{AB}$ depends on the frequencies of the alleles in questions and are therefore not always convinient.
Therefore, LD between two loci is commonly measured in two different ways. 
That is $D'$ and $r^2$.
\citet{Lewontin1964} suggested to use
\begin{equation}\label{eq:dprime}
  D' = D/D_{\min}
\end{equation}
where 
\begin{equation*}
  D_{\min}= \begin{cases}
    \max\{-p_A p_B,\,-(1-p_A)(1-p_B)\} & \text{when } D < 0\\
    \min\{p_A (1-p_B),\,(1-p_A) p_B\} & \text{when } D > 0
  \end{cases} 
\end{equation*}
Alternativly, one can also use the correaltion coefficient between the two loci 
\begin{equation}\label{eq:r2}
  r=\frac{D}{\sqrt{p_A(1-p_A)p_B (1-p_B)}}
\end{equation}
An important consequence for assocation studies is that an assocation between a trait and an allele is unlikely to be the actual causal SNP\@.
An association between an SNP and a trait can arise out of multiple reasons.
First, the association could represent the true effect and the particular SNP has a causal relationship to the trait in question.
Second and more likely, the associated SNP is a proxy of the causal SNP since both are in high \acrfull{ld}.
Third, the association is random fluctioation within the sample, and fourth the assocation is due to confounding errors such as population stratification or genotyping errors.

During the last decade the cost of obtaining a person's genome reduced rapidly.
This enabled the analysis of not only a few genetic markers but the whole genome.
This analysis is called \acrfull{gwas}.
Most GWAS are population based studies of unrelated samples and analyse SNPs with a substential frequency within the population, usually $\ge 1\%$.
Here it is important to note that all samples have distante relatives and the term `unrelated' is used loosly.
In fact it becomes increasingly difficult to establish the exact genetic relationship between two participants by SNPs alone as further those subjects are appart within the family tree.

The assocation between an SNP and a trait can be expressed as a linear regression model for a continious and binary phenotype respectivly
While other methods to estiamte the effect exist, such as the $\chi^2$-test, a linear regression model allows to include multiple covariates.
The inclusion of multiple covariates is crucial since it allows to adjust for confounding factors such as population stratification, sex, genotyping platforms and others.
I will now proceed with describeing one of the most important confounder, population stratification.

\subsubsection{Population Stratification}
\label{ssub:population_stratification}
Population stratification takes place when differences in the frequencies of alleles among cases and controls are not due to an causal relationship between the SNP and the trait.
Rather it is caused by ancestral differences across populations.
An association is affected by population stratification if the trait is more prevalent in one population while the allele frequencies vary across the populations as well.

Commonly one can adjust for population stratification by the usage of \acrfull{pca}.
PCA is a procedure which transforms a set of correlated variables into a set of linear uncorrelated once, called principle components (\acrshort{pc}).
The number of PC can be smaller or equal that of the number of intial variables and the first PC accounts for most of the variablity in the set of correlated variables.
Each following PC explains the most variance constrained that it is othogonal to the previous.
Using PCA on a matrix of genotypes results in a set of PC which explain the genetic variation within the sample.
Given the sample is a mixture out of multiple population with different ancestry the computated PC will often have geographic interpretation.
Therefore including PCs into the association model will adjust for population stratification arising due to differences in allele frequencies and disease prevalence.

\subsubsection{Multiple Testing}
\label{ssub:multiple_testing}
By testing a large amount of SNP one would expect a number of significant assocation simply by chance.
Therefore one needs to adjust the significant threshold.
For example, one could simply adjust the significant threshold by the numver of tests performed, also called Bonferroni threshold.
However, this would result in an overly conservative threshold and in a number of false negative associations~\cite{Benjamini1995}.
Alternativly~\citet{Benjamini1995} developed the \acrfull{fdr} a more powerful multiple testing correction. 
However, a further complication in genetics assocation studies arises from LD\@.
Due to correlation among sets of SNPs computated test statistics are not independent.
This requires to consinder correlation among SNPs when deciding for a multiple testing threshold.
As a result~\citet{Dudbridge2008} estiamted a significant threshold of $7.2 \times 10^{-8}$ by analysing a large set of genotyped samples and their corresponding LD\@.
Nevertheless, most assocation studies today use a threshold of $5\times 10^{-8}$. %TODO needs explanation and citation

\acrshort{gwas} are usefull in identifying molecular markers for traits and diseases.
While there are multiple possible confounding factors, such as population stratification, methods have been developed to approach these problems successfully.
The process of conducting a GWAS is nowdays well defined.
However, usually GWAS do not deal with genetic variations with a frequencies below 1\%.
Hence I will next give an overview of assocation studies of rare variants.

\subsection{Association studies on rare varitants}
\label{sub:association_studies_on_rare_varitants}

\subsection{Heritability and Genetic Correlation}
\label{sub:heritability_and_genetic_correlation}

As already described above, studies on twins were able to assess the heritability of traits by consindering the inter-correlation among MZ and DZ twins.
Therefore, one would expect that the variance explained by all SNPs combined would result in similar estimtes.
Unfortunatly, for many traits, this is not the case.
This is called the `Missing Heritability Problem'~\cite{Vineis2010}.
A number of reasons have been suggested for the discrepancy between estimates in twin stuides and those in GWAS\@.
First, the assumption in twin studies might be violated and estiamtes are too high.
Second, GWASs only consinder common variants and rare variants might account for the missing heritability.
Third, epistasis is only insufficiently captured in most studies and could account for the differences.
Fourth, gene-enviornment interaction could also explain partly the differences between the two types of studies.
While a single reason for the missing heritability seems unlikely it is still an ongoing research objective to account for the differences.
Further, the estiamtion of heritability from genotyped data is not trivial.
Several methods have been proposed in the past, most noteably \acrfull{gcta} and LD-score regression.
GCTA uses mixed linear model to the fit the effect of all SNPs by making use of the genetic relationship matrix of all included subjects~\cite{Yang2011}.
If $\textbf{A}$ is the genetic relatedness matrix then
\begin{equation}
  y = X\beta + g + \epsilon \text{ with } var(y) = V = A\sigma^2_g + I\sigma^2_\epsilon
\end{equation}
in which $y$ is the phenotype and $\beta$ are the estimated effect sizes of all covariates and the total genetic effects for each individual $g$ is $g \sim N(0, A\sigma^2_g)$.
GCTA is then able to estiamte the variance explained by all SNPs $\sigma^2_g$ by restircted maximum likelihood.
Further one can extent this model to bivariate linear mixed models to estiatme the genetic correlation between two traits.
If $y_1 = X_1\beta_1 + g_1 + \epsilon_1$ for trait 1 and $y_2= X_2\beta_2 + g_2 + \epsilon_2$ then the variance-covariance matrix $V$ is
\begin{equation}
  V = 
  \begin{pmatrix}
    Z_1AZ_1'\sigma_{g1} + I\sigma^2_{\epsilon 1} & Z_1AZ_2'\sigma_{g_1g_2} \\
    Z_2AZ_1'\sigma_{g_1g_2} & Z_2AZ_2'\sigma_{g2} + I\sigma^2_{\epsilon 2}
  \end{pmatrix}
\end{equation}
in wich $X$ and $Z$ are the incidence matrices for the effects of $\beta$ and $g$.
However, GCTA requires considerable computation power as well as the availability of the raw genotyped data.
Hence LD-score regression has been develped to estiamte heritability on summary statistics only.

LD-score regression makes use to the previously outlined LD among tagged and causal SNPs.
Test statistics of SNPs in high LD with the causal variant will be elevated proportioal to their LD\@.
Thus the more genetic variation an SNP tagges the higher the probability that it will tag a causal variant.
LD-score regression makes use of this relationship and regresses the estimated $\chi^2$ from the association study on the LD-score, which measures the overall LD of variant $j = 1, \ldots, M$ as $\ell_j = \sum^M_{k=1} r^2_{jk}$. 
The slope of this regression can then be interpretated as an estimate of heritability~\cite{Bulik-Sullivan2015}.
Similar if we replace the $\chi^2$ of a single study by the product of the z-score of two seperate studies and regress it onto $\ell_j \sqrt{N_{1j}N_{2j}}$ the slope can be interpreted as the genetic covariance between trait 1 and 2~\ref{BulikSullivan2015a}.

These develpments have enabled rescent research to uncover the inter-correlation among a variety of different traits.
However, genetic correlation can arise from a multitude of different sources.
Figure~\ref{fig:genetic_correlation} shows 4 different ways genetic correlation can arise.
First and formost, genetic correaltion can arrise if two traits are caused by the same genetic variant (see Figure~\ref{fig:pleiotropy}).
Second, a genetic factor which causes phenotype 1 can in turn cause phenotype 2 (see mediated pleiotropy in Figure~\ref{fig:mediated_pleiotropy}).
Third, genetic correlation can also arrise from Assortative mating as shown in Figure~\ref{fig:assortative_mating}.
Assortative mating is the process of non-random mating within a population.
Specifically, some traits might be more desirable for the opposite sex.
For example, consider two traits which share no causal variant.
Trait 1 is desirable in male while trait 2 is desirable in female.
Over a few genertation this will result in LD between causal variants of trait 1 and 2 despite sharing no initial causal SNPs. 
Last, also parental effect can result in genetic corrections as displayed in Figure~\ref{fig:parental_effects}.
Specifically, genetic components which cause trait $1$ in the parents might influence the child enviornment which in turn results in trait 2. 
Nevertheless the genetic correlations estiamted by LD-score and GCTA are unable to distinguies between the possible sources of genetic correlations.

\begin{figure}[htp]
  \begin{subfigure}{0.5\textwidth}
    \resizebox{0.8\linewidth}{!}{\input{introduction/figure/genetic_correlation/pleiotropy.tex}} 
    \caption{Pleiotropy}\label{fig:pleiotropy}
  \end{subfigure}
  \begin{subfigure}{0.5\textwidth}
    \resizebox{0.8\linewidth}{!}{\input{introduction/figure/genetic_correlation/mediated_pleiotropy.tex}} 
    \caption{Mediated Pleiotropy}\label{fig:mediated_pleiotropy}
  \end{subfigure}
  \begin{subfigure}{0.5\textwidth}
    \resizebox{\linewidth}{!}{\input{introduction/figure/genetic_correlation/assortative_mating.tex}} 
    \caption{Assortative Mating}\label{fig:assortative_mating}
  \end{subfigure}
  \begin{subfigure}{0.5\textwidth}
    \resizebox{\linewidth}{!}{\input{introduction/figure/genetic_correlation/parental_effects.tex}}
    \caption{Parental Effects}\label{fig:parental_effects}
  \end{subfigure}
  \caption{Different sources of genetic correlation according to~\citet{Pickrell2016}}\label{fig:genetic_correlation}
\end{figure}

To conclude, during the last few years large gains have been made in fostering our understanding of heritability and genetic correlations with the development of GCTA and LD-score.
While these tools are able to estimate heritability and genetic correlations of a number of traits it remains difficult to distiguish between different sources of genetic correlations.


%In contrast, the narrow sense heritability only contains the accumulated effect of all genetic variants.
%In contrast the narrow sense heritability can also be derived when using adopted siblings.
%If $r_{sib}$ is the correlation among siblings raised together and $r_{ad}$ the correlation of those raised appart then $h^2_{narrow} = r_{sib} - r_{ad}$.
